github/ci/prow-deploy/kustom/base/configs/current/labels/labels.yaml

```diff
@@ -215,6 +215,11 @@ default:
       previously:
         - name: topic/virtualization
           color: c5def5
+    - name: sig/code-quality
+      color: c5def5
```

Comment:
this is peanuts, but using a different color code would be helpful.

Reply:
just suggest a number and I'll happily change.

Reply:
[ce23cf](https://encycolorpedia.com/ce23cf) => a shade of magenta. Because <insert any justification ...> . 

I just wanted something different than the "regular" "serious" sigs.

Reply:
Done